libbpf-tools/Makefile

```diff
@@ -111,6 +123,22 @@ $(OUTPUT)/%.bpf.o: %.bpf.c $(LIBBPF_OBJ) $(wildcard %.h) $(ARCH)/vmlinux.h | $(O
 		     -I$(ARCH)/ $(INCLUDES) -c $(filter %.c,$^) -o $@ &&      \
 	$(LLVM_STRIP) -g $@
 
+ifdef ENABLE_MIN_CORE_BTFS
+
+.PHONY: update_btfhub
+update_btfhub: $(BTF_HUB_ARCHIVE)
+	$(Q)cd $(BTF_HUB_ARCHIVE) && git pull
+
+$(BTF_HUB_ARCHIVE):
+	$(call msg,GIT CLONE,$@)
+	$(Q)git clone https://github.com/aquasecurity/btfhub-archive/ $(BTF_HUB_ARCHIVE)
+
+BPF_O_FILES = $(patsubst %,$(OUTPUT)/%.bpf.o,$(APPS))
```

Comment:
given this definition is used in one place in one rule, let's just inline it (at least for now)?